libraries/chain/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ file(GLOB HEADERS "include/eosio/chain/*.hpp"
                   "include/eosio/chain/webassembly/*.hpp"
                   "${CMAKE_CURRENT_BINARY_DIR}/include/eosio/chain/core_symbol.hpp" )
 
+option(EOSIO_ENABLE_DEVELOPER_OPTIONS "enable developer options for EOSIO" OFF)
```

Comment:
I thought when you said you were going to move this, you were going to put it in the root CMakeLists.txt.

This doesn't belong in the chain folder because it doesn't actually affect libchain and instead affects the nodeos as a whole (specifically the `chain_plugin`). The corresponding `target_compile_definitions` will also need to moved to `plugins/chain_plugin/CMakeLists.txt` and changed to affect the `chain_plugin` target.

Reply:
Ahah sorry. It was wrongly placed before because it was wrapped within the conditional `if("eos-vm-oc" IN_LIST EOSIO_WASM_RUNTIMES` which was obviously completely wrong! So I moved it to be free on the conditional.

Sure, I'll move that to `chain_plugin`.

Reply:
So just to be sure I get this right.

The `option` definition should go on the root `CMakeLists.txt` and the `target_compile_definitions` in `plugins/chain_plugin/CMakeLists.txt`, correct?

Reply:
Yes.

Reply:
Done.